build.gradle

```diff
@@ -195,7 +197,7 @@ repositories {
 }
 
 dependencies {
-
+    compile group: 'org.meanbean', name: 'meanbean', version: '2.0.3'
```

Comment:
MeanBean automatically unit tests all setter and getters. Basically I have added a meanBean utility to get setter/getter coverage if any class extends AbstractEntiryTest. This will save efforts in future and helps in increasing PI testing.